src/widgets/history/HistoryComparison.php

```
@@ -79,57 +75,20 @@ public function hasDifferences(): bool
-                'mail',
-            ),
-            self::integerMetric(
-                'Excessive DB callers',
-                $baseline->excessiveCallersCount,
-                $target->excessiveCallersCount,
-                'db',
-            ),
-        ];
+        $metrics = [];
+
+        foreach (SummaryMetricComparison::between($baseline, $target) as $metric) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Debug\Comparison\SummaryMetricComparison' in method 'buildMetrics'. (undefined)

(StaticAccess)